Source/missiles.cpp

```diff
@@ -1980,8 +1971,6 @@ void AddFireball(Missile &missile, Point dst, Direction midir)
 	missile._mirange = 256;
 	missile.var1 = missile.position.start.x;
 	missile.var2 = missile.position.start.y;
-	missile.var4 = missile.position.start.x;
-	missile.var5 = missile.position.start.y;

```

Comment:
Similar to the other one, can you elaborate why this is a safe removal?

Reply:
Never used by `MI_Fireball`